frontend/app/src/components/helper/PrioritizedList.vue

```diff
@@ -103,59 +103,58 @@ const autoCompleteHint: ComputedRef<string> = computed(() => {
 
 <template>
   <div>
-    <VSheet outlined rounded>
-      <div class="pb-0" :class="slots.title ? 'pa-4' : 'pa-0'">
-        <div v-if="slots.title" class="text-h6 pb-4">
+    <RuiCard rounded="md" class="[&>div]:!p-0">
```

Comment:
we can probably add a prop to card to not add padding (some libs use `flush`).
doesn't affect your PR though.

Reply:
Yeah that sounds good. We can do that for next release